### PR TITLE
Add direct HTML input support

### LINKED ISSUE #13: Allow direct HTML input
https://github.com/sindresorhus/capture-website/issues/3 

https://github.com/sindresorhus/capture-website/pull/33

### PR DESCRIPTION
**Pull request by fisker**
fixes #13

`stdin` support is not added yet https://github.com/sindresorhus/capture-website-cli/issues/13#issuecomment-538448960

### CODE REVIEW

**Comment by fisker** [comment]
ready

File: readme.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ Note to Linux users: If you get a "No usable sandbox!" error, you need to enable
...
   Usage
-    $ capture-website <url|file> [output-file]
-
-    The screenshot will be written to stdout if there's no output file argument
+    $ capture-website <url|file>
+    $ echo "<h1>Unicorn</h1>" | capture-website
 
   Options
+    --output                 Image filename, if omitted, screenshot will be written to stdout
+    --html                   Set input type to \`html\`  [default: false]
```
Comment (sindresorhus) [inline readme.md]: This should not be here

File: test.js [hunk 2]
```diff
@@ -17,9 +17,18 @@ test('main', async t => {
 	await server.close();
 });
 
+test('support HTML input', async t => {
+	const {stdout} = await execa('./cli.js', [], {
+		input: '<h1>Unicorn</h1>',
+		encoding: 'buffer'
```
Comment (sindresorhus) [inline test.js]: Huh? It's a string.

File: cli.js [hunk 3]
```diff
@@ -4,14 +4,15 @@ const meow = require('meow');
...
 const cli = meow(`
 	Usage
-	  $ capture-website <url|file> [output-file]
-
-	  The screenshot will be written to stdout if there's no output file argument
+	  $ capture-website <url|file>
+	  $ echo "<h1>Unicorn</h1>" | capture-website
 
 	Options
+	  --output                 Image filename, if omitted, screenshot will be written to stdout
```
Comment (sindresorhus) [inline cli.js]:
```suggestion
	  --output                 Image file path (writes it to stdout if omitted)
```

File: test.js (hunk 2 above)
Comment (fisker) [reply test.js]: I'm confused, is this `encoding` for `stdout`?

https://github.com/sindresorhus/execa#encoding

Comment (sindresorhus) [reply test.js]: Oops, yes.

**Comment by sindresorhus** [comment]
Looks good. Thanks :)

**Pull request merged**